daemon/algod/server.go

```diff
@@ -142,17 +142,17 @@ func (s *Server) Initialize(cfg config.Local, phonebookAddresses []string, genes
 
 	// if we have the telemetry enabled, we want to use it's sessionid as part of the
 	// collected metrics decorations.
-	fmt.Fprintln(logWriter, "++++++++++++++++++++++++++++++++++++++++")
-	fmt.Fprintln(logWriter, "Logging Starting")
+	s.log.Infoln("++++++++++++++++++++++++++++++++++++++++")
+	s.log.Infoln("Logging Starting")
```

Comment:
Seems weird to print "Logging Started" in the log file.

Some better options:
* Write "Logging Started: <logfile>" to stdout.
* Delete these lines.

Either way, definitely remove the `++++++++++++++++++++++++++++++++++++++++` banner.

Reply:
+++ATH0

Reply:
If I'm restarting a container, and tailing logs, it is helpful to see a clear location where the restart is obvious. But that said, it shouldn't deviate from the normal formatting expected by machine parsers, e.g. Filebeat.

Reply:
Maybe, "algod process start"

Reply:
> +++ATH0

Is this https://support.usr.com/support/3cxm756/3cxm756-ug/atcoms.htm ?